pages/facts.html

```diff
@@ -4,9 +4,88 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <title>Bidet Facts</title>
+  <link href="/styles/style.css" rel="stylesheet"> 
+  <link rel="stylesheet" href="/styles/facts.css"> 
 </head>
+<nav>
+  <a href="index.html">Homepage</a>
+  <a href="gallery.html">Portraits of Bidets</a>
+  <a href="facts.html">Everything You Need To Know About Bidets</a>
+</nav>
+
 <body>
+  <section>
+  <h1> Everything You Need To Know About Bidets </h1>
+    <p> </p>
+  </section>
```

Comment:
This `section` could be a `header`! 😄 Also, if you want spacing beneath the `h1`, you can use `margin` instead of an empty paragraph.